src/NotificationEventMailing.php

```diff
@@ -263,6 +263,7 @@ public static function send(array $data)
                     }
                 } else {
                     $inline_docs = [];
+                    $current->fields['body_html'] = html_entity_decode($current->fields['body_html']);
```

Comment:
This will transform `<`, `>` and `&` from text or code snippets into their unencoded form and potentially breaks the rendered HTML.